charts/prometheus-blackbox-exporter/templates/_helpers.tpl

```diff
@@ -150,10 +150,10 @@ securityContext:
 {{- end }}
 {{- with .Values.extraInitContainers }}
 initContainers:
-  {{- toYaml . | indent 2 }}
+{{ toYaml . | indent 2 }}
```

Comment:
```suggestion
{{- toYaml . | indent 2 }}
```

Reply:
After apply the suggestion, it will fail to generate the template.
```
$ cat > test.patch << EOF
--- a/prometheus-blackbox-exporter/templates/_helpers.tpl
+++ b/prometheus-blackbox-exporter/templates/_helpers.tpl
@@ -150,7 +150,7 @@
 {{- end }}
 {{- with .Values.extraInitContainers }}
 initContainers:
-  {{- toYaml . | indent 2 }}
+{{- toYaml . | indent 2 }}
 {{- end }}
 containers:
 {{- with .Values.extraContainers }}
EOF
$ cat > values.yaml << EOF
extraInitContainers:
- name: k8s-sidecar
  image: quay.io/kiwigrid/k8s-sidecar
extraContainers:
- name: k8s-sidecar
  image: quay.io/kiwigrid/k8s-sidecar
EOF
$ helm pull --untar prometheus-community/prometheus-blackbox-exporter
$ patch -p1 < test.patch
$ helm template -f values.yaml prometheus-blackbox-exporter/
Error: YAML parse error on prometheus-blac-kbox-exporter/templates/deployment.yaml: error converting YAML to JSON: yaml: line 37: block sequence entries are not allowed in this context

Use --debug flag to render out invalid YAML
$ helm template -f values.yaml prometheus-blackbox-exporter/ --debug
...
    spec:
      automountServiceAccountToken: false
      serviceAccountName: release-name-prometheus-blackbox-exporter
      restartPolicy: Always
      initContainers:  - image: quay.io/kiwigrid/k8s-sidecar
          name: k8s-sidecar
      containers:- image: quay.io/kiwigrid/k8s-sidecar
        name: k8s-sidecar
```

Reply:
Sorry about that.